grpc-google-cloud-language-v1/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <!-- TODO: remove after 1.99.3 released -->
+    <differenceType>6001</differenceType>
+    <className>com/google/cloud/language/v1/*Grpc</className>
+    <field>METHOD_*</field>
```

Comment:
can this be more specific?

Reply:
We could explicitly list out every removed field, but it seems unnecessary extra work. These files are complete generated and we know this only affects fields prefixed by `METHOD_`